explainer.md

```diff
@@ -72,6 +72,113 @@ There are still many open questions, but a few big ones:
 - Will existing protocols ([example](https://github.com/openid/OpenID4VP/issues/125)) adopt this API?
 - Will regulation ([example](https://digital-strategy.ec.europa.eu/en/library/european-digital-identity-architecture-and-reference-framework-outline)) adopt this API?
 
+# [Self-Review Questionnaire: Security and Privacy](https://w3ctag.github.io/security-questionnaire/)
+
+> 01.  What information does this feature expose,
+>      and for what purposes?
+
+The Digital Credential API exposes a one-time user-mediated end-to-end encrypted communication channel from websites, to digital wallet applications, and back to websites. It is designed to be a better option than established lower-level communication channels like custom schemes, QR codes, and server-to-server network communication. How exactly this channel is used is up to the wallet applications and host operating system, but we are designing it to be suitable for conveying presentations of digital credentials such as claims in mobile driver’s licenses.
```

Comment:
"The Digital Credential API exposes a one-time user-mediated end-to-end encrypted communication channel from websites, to digital wallet applications, and back to websites." is the sentence that jumped out at me @RByers FWIW

Reply:
Yes thank you @bc-pi, I agree that was misleading. I corrected it to "with end-to-end encryption of the response".